inst/rmarkdown/templates/ams/skeleton/skeleton.Rmd

```diff
@@ -1,62 +1,72 @@
 ---
-journal: jhm
-layout: draft
+twocol: FALSE  #  Use FALSE for submission. 
 title: Title here
-author1: Andrew N. Other
-author2: Fred T. Secondauthor
-currentaddress: "Current address: Some other place, Germany"
+author1: 
+  name: Author One 
+  email: email adress
+author2: 
+  name: Author Two 
+  affiliation: "Current affiliation: NCAR, Boulder, Colorado."
 affiliation: "American Meteorological Society,Boston, Massachusetts"
-exauthors: 
-  - name: Ping Lu
-    exaffiliation: Princeton University
-    correspondingauthor: "American Meteorological Society, 45 Beacon St., Boston, MA 02108."
-    email: \email{groupleader@unknown.uu}
-  - name: Miao Yu
-    exaffiliation: University of Waterloo
-    currentaddress: "Current address: Some other place, Canada"
+extra_authors: 
+  - name: Extra Author
+    affiliation: Affiliation, City, State/Province, Country 
 abstract: |
-  Enter the text of your abstract here.  This is a sample American Meteorological Society (AMS) \LaTeX\ template.  This document provides authors with instructions on the use of the AMS \LaTeX\ template.  Authors should refer to the file amspaper.tex to review the actual \LaTeX\ code used to create this document. The template.tex file should be modified by authors for their own manuscript.
-bibliography: amstest.bib
-csl: american-meteorological-society.csl
+  Enter the text of your abstract here. This is a sample American Meteorological Society (AMS) \LaTeX\ template.  This document provides authors with instructions on the use of the AMS \LaTeX\ template.  Authors should refer to the file amspaper.tex to review the actual \LaTeX\ code used to create this document. The template.tex file should be modified by authors for their own manuscript.
+statement: |
+  An optional plain language summary suitable for wider readership that provides additional context for why the work is relevant to science and society. It should be concise (≤120 words) and authors should follow the guidelines for significance statements (\url{https://www.ametsoc.org/index.cfm/ams/publications/authors/journal-and-bams-authors/significance-statements/}) (currently MWR,WAF, and WCAS only).
+capsule: |
+   A short (20-30 words maximum) capsule summary of your article (for BAMS only).
+bibliography: references.bib
 output: rticles::ams_article
```

Comment:
change here to pass the check.

```
output: 
  rticles::ams_article:
    latex_engine: xelatex
```

Reply:
Does it means this format has to default to xelatex ?

Reply:
Just for this template. It seems the bib file contain unicode character which will not map to output through pdflatex. Such issue blocked the check. Since the bib file might be from ASM, we could set the default engine to xelatex to be friendly to the user.